toolbox/fdc3-for-web/demo/src/client/apps/app6.ts

```diff
@@ -13,7 +13,11 @@ getAgent().then(async fdc3 => {
     },
   });
 
-  log!!.textContent = `Got resolution: ${JSON.stringify(reso)}`;
-  const result = await reso.getResult();
-  log!!.textContent += `Got result: ${JSON.stringify(result)}`;
+  if (log) {
+    log.textContent = `Got resolution: ${JSON.stringify(reso)}`;
+    const result = await reso.getResult();
+    log.textContent += `Got result: ${JSON.stringify(result)}`;
+  } else {
+    console.error("Unable to load resolution as the log element didn't exist!");
```

Comment:
I guess this is a stylistic choice, but I would always prefer to use ! over if() in this scenario, since the log element will always be set.  Adding the error() is just noise that will never get called.